addons/website/views/snippets/s_intro_pill.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_intro_pill" name="Intro Pill">
+    <section class="s_intro_pill pt40 pb40">
+        <div class="container-fluid">
+            <div class="row o_grid_mode" data-row-count="11">
+                <div class="o_grid_item oe_img_bg o_grid_item_image g-col-lg-5 g-height-9 col-lg-5 order-lg-0" style="order: 2; z-index: 1; grid-area: 1 / 1 / 10 / 6; --grid-item-padding-y: 0px; --grid-item-padding-x: 0px;">
+                    <img class="img-fluid mx-auto" data-shape="web_editor/geometric_round/geo_round_pill" data-file-name="s_intro_pill_default_image.jpg"
+                    data-original-mimetype="image/jpeg" src="/web_editor/image_shape/website.s_intro_pill_default_image/web_editor/geometric_round/geo_round_pill.svg"/>
+                </div>
+                <div class="o_grid_item g-height-11 g-col-lg-8 col-lg-8 order-lg-0" data-name="Box" style="order: 1; z-index: 3; grid-area: 1 / 3 / 12 / 11; --grid-item-padding-y: 120px; text-align: center;">
```

Comment:
When I try to recreate it with the editor, the `text-align` is on `h1` and `p`.

Reply:
Indeed, fixed 👍